helpers/polizasHelper/polizashelper.go

```diff
@@ -0,0 +1,260 @@
+}
+
+// GetElementosEntrada obtiene todos las ActasId que sean entradas (P8)
+//
+func GetIdsActasEntrada() (IdsActasEntradas []int, outputError map[string]interface{}) {
+
+	defer func() {
+		if err := recover(); err != nil {
+			outputError = map[string]interface{}{
+				"funcion": "GetIdsActasEntrada - Unhandled Error!",
+				"err":     err,
+				"status":  "500",
+			}
+			panic(outputError)
+		}
+	}()
+
+	var (
+		RespuestaAPI     interface{}
+		Movimientos      []models.Movimiento
+		ActasIdsEntradas []int
+	)
+
+	//Traer el detalle de las entradas (json)
+	params := url.Values{}
+	params.Add("query", "Activo:true,Detalle__contains:consecutivo\": \"P")
+	params.Add("fields", "Detalle")
+	params.Add("limit", "-1")
+	urlRespuestaAPI := "http://" + beego.AppConfig.String("movimientosArkaService") + "movimiento?" + params.Encode()
+	if _, err := request.GetJsonTest(urlRespuestaAPI, &RespuestaAPI); err != nil {
+		logs.Error(err)
+		outputError = map[string]interface{}{
+			"funcion": "GetIdsActasEntradas - request.GetJsonTest(urlRespuestaAPI, &RespuestaAPI)",
+			"err":     err,
+			"status":  "502",
+		}
+		return nil, outputError
+	}
+
+	// Asignar a Movimientos RespuestaAPI
+	if err := formatdata.FillStruct(&RespuestaAPI, &Movimientos); err != nil {
+		logs.Error(err)
+	}
+
+	//Itera los Movimientos para poder guardar solo las actas de recibido
+	for _, movimiento := range Movimientos {
+		var detalle map[string]interface{}
+		if err := json.Unmarshal([]byte(movimiento.Detalle), &detalle); err != nil {
+			logs.Error(err)
+		}
+		ActasIdsEntradas = append(ActasIdsEntradas, int(detalle["acta_recibido_id"].(float64)))
+	}
+
+	//Remueve los Ids replicados
+	IdsActasEntradas = removeDuplicateIds(ActasIdsEntradas)
+
+	return
+}
+
+// Obtiene todos los idSubgrupos que necesitan poliza
+//
+func GetSubgruposPoliza() (IdSubgruposPoliza []int, outputError map[string]interface{}) {
+
+	defer func() {
+		if err := recover(); err != nil {
+			outputError = map[string]interface{}{
+				"funcion": "SubgruposPoliza - Unhandled Error!",
+				"err":     err,
+				"status":  "500",
+			}
+			panic(outputError)
+		}
+	}()
+
+	var (
+		Respuesta         interface{}
+		DetalleSubgrupo   []models.DetalleSubgrupo
+		IdDetalleSubgrupo []int
+	)
+
+	//Traer el detalle de las entradas (json)
+	params := url.Values{}
+	params.Add("query", "TipoBienId__NecesitaPoliza:True,Activo:True")
+	params.Add("limit", "-1")
+	urlSubgrupos := "http://" + beego.AppConfig.String("catalogoElementosService") + "detalle_subgrupo?" + params.Encode()
+	if _, err := request.GetJsonTest(urlSubgrupos, &Respuesta); err != nil {
+		logs.Error(err)
+		outputError = map[string]interface{}{
+			"funcion": "GetSubgruposPoliza - request.GetJsonTest(urlSubgrupos, &Respuesta)",
+			"err":     err,
+			"status":  "502",
+		}
+		return nil, outputError
+	}
+
+	// Asignar a DetalleSubgrupo RespuestaAPI
+	if err := formatdata.FillStruct(&Respuesta, &DetalleSubgrupo); err != nil {
+		logs.Error(err)
```

Comment:
Revisar: Si el `err`or retornado por `formatdata.FillStruct()` **no** es crítico, bajar el nivel de logs:
```suggestion
		logs.Warn(err)
```
En otro caso, falta el manejo de este error, armando y retornando un error formateado